deployment/aws/infrastructure/demo/main/gh.tf

```diff
@@ -73,13 +73,23 @@ data "aws_iam_policy_document" "gh_actions_policy_document" {
     actions = [
       "ecs:DescribeServices",
     "ecs:UpdateService"]
-    resources = [aws_ecs_service.admin_app_service.id, aws_ecs_service.backend_service.id, aws_ecs_service.nginx_service.id]
+    resources = [
+      aws_ecs_service.admin_app_service.id,
+      aws_ecs_service.backend_service.id,
+      aws_ecs_service.nginx_service.id,
```

Comment:
Are we still using nginx? I though we moved to Caddy?

Reply:
Not for AWS deployment!